web/text_layer_builder.css

```diff
@@ -108,12 +108,16 @@
     display: block;
     position: absolute;
     inset: 100% 0 0;
-    z-index: -1;
+    z-index: 0;
```

Comment:
Why this change ?

Reply:
Without this change, `.endOfContent` is below `.textLayer` so, when dragging the selection over an empty space, Firefox's "collision detection" finds `.textLayer` rather than `.endOfContent`, and selects everything from the start of `.textLayer`.

Reply:
The z-index change avoids the floating highlight button to be clickable:
![image](https://github.com/mozilla/pdf.js/assets/5641725/764b9e2b-8dc9-40a6-b17e-5c37c13a59c2)

Reply:
Whops, I made sure that it worked, then changed how I was applying the z-index, and forgot to re-check. I'll see how to add a test for this too.